core/src/main/scala/cats/std/either.scala

```diff
@@ -23,7 +23,7 @@ trait EitherInstances extends EitherInstances1 {
         }
     }
 
-  implicit def eitherInstances[A]: Monad[Either[A, ?]] with Traverse[Either[A, ?]] =
+  implicit def catsMonadForEither[A]: Monad[Either[A, ?]] with Traverse[Either[A, ?]] =
```

Comment:
For this and several other instances you've somewhat arbitrarily picked the first first type class when an instanced provides several type classes. This may not be a big deal and I don't think I really have a better suggestion, but it does feel a little off.